Helper/FeatureSwitch/Definitions.php

```diff
@@ -38,12 +38,24 @@ class Definitions {
      */
     const M2_SAMPLE_SWITCH_NAME = "M2_SAMPLE_SWITCH";
 
+    /**
+     * Whether bolt is enabled. This will be used for slow rollout / controlling bolt
+     * from the backend.
+     */
+    const M2_BOLT_ENABLED = "M2_BOLT_ENABLED";
+
     const DEFAULT_SWITCH_VALUES = array(
         self::M2_SAMPLE_SWITCH_NAME =>  array(
           self::NAME_KEY            => self::M2_SAMPLE_SWITCH_NAME,
           self::VAL_KEY             => true,
           self::DEFAULT_VAL_KEY     => false,
           self::ROLLOUT_KEY         => 0
-      )
+        ),
+        self::M2_BOLT_ENABLED =>  array(
+          self::NAME_KEY            => self::M2_BOLT_ENABLED,
+          self::VAL_KEY             => true,
+          self::DEFAULT_VAL_KEY     => false,
+          self::ROLLOUT_KEY         => 100
```

Comment:
Shouldn't this be 0 by default? Otherwise, we always override with feature switches the value of the enabled/disabled button in the magento admin settings, right?

Reply:
no we dont want this to be 0 by default. we want bolt to always be enabled and we can choose to disable when needed.

the reason is: 
- we have ~100 merchants already live. We dont want to have to go add voerrides for each of them
- When a merchant is added we can choose to slow roll out for them if needed, and then we reduce this,. but by default we ship fully.

LMK if that makes sense

Reply:
@roopakv and I synced offline - I had a misunderstanding on rollout% - I interpreted this as a % chance of checking the status of the feature flag, whereas it's actually the state of the flag itself. @roopakv will update the phpdoc to be more explicit.